src/settings-ui/Settings.UI/OOBE/Views/OobeWhatsNew.xaml

```diff
@@ -1,77 +1,81 @@
-      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-      xmlns:local="using:Microsoft.PowerToys.Settings.UI.OOBE.Views"
-      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-      xmlns:controls="using:Microsoft.PowerToys.Settings.UI.Controls"
-      xmlns:toolkitcontrols="using:CommunityToolkit.WinUI.UI.Controls"
-      mc:Ignorable="d"
-      Loaded="Page_Loaded">
+﻿<Page
+    x:Class="Microsoft.PowerToys.Settings.UI.OOBE.Views.OobeWhatsNew"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.PowerToys.Settings.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Microsoft.PowerToys.Settings.UI.OOBE.Views"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:toolkitcontrols="using:CommunityToolkit.WinUI.UI.Controls"
+    Loaded="Page_Loaded"
+    mc:Ignorable="d">
 
 
     <Grid Margin="0,24,0,0">
         <Grid.RowDefinitions>
-            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
 
-        <StackPanel Grid.Row="0" Margin="32,0,0,0" Orientation="Vertical" VerticalAlignment="Top">
+        <StackPanel
+            Grid.Row="0"
+            Margin="32,0,0,0"
+            VerticalAlignment="Top"
+            Orientation="Vertical">
             <TextBlock
                 x:Uid="Oobe_WhatsNew"
                 AutomationProperties.HeadingLevel="Level1"
                 Style="{StaticResource TitleTextBlockStyle}" />
             <HyperlinkButton
-                NavigateUri="https://github.com/microsoft/PowerToys/releases"
                 Margin="0,0,0,16"
+                NavigateUri="https://github.com/microsoft/PowerToys/releases"
                 Style="{StaticResource TextButtonStyle}">
-                <TextBlock
-                    x:Uid="Oobe_WhatsNew_DetailedReleaseNotesLink"
-                    TextWrapping="Wrap" />
+                <TextBlock x:Uid="Oobe_WhatsNew_DetailedReleaseNotesLink" TextWrapping="Wrap" />
             </HyperlinkButton>
         </StackPanel>
         <InfoBar
-                Severity="Error"
-                x:Name="ErrorInfoBar"
-                Grid.Row="1"
-                x:Uid="Oobe_WhatsNew_LoadingError"
-                Visibility="Collapsed"
-                VerticalAlignment="Top"
-                IsClosable="False"
-                IsOpen="True"
-                IsTabStop="True" />
+            x:Name="ErrorInfoBar"
+            x:Uid="Oobe_WhatsNew_LoadingError"
+            Grid.Row="1"
+            VerticalAlignment="Top"
+            IsClosable="False"
+            IsOpen="True"
+            IsTabStop="True"
+            Severity="Error"
+            Visibility="Collapsed" />
```

Comment:
I think Visibility and IsOpen are confilcting properties // Visibility is unneeded

Reply:
Fixed!